test-server/test-echo.c

```diff
@@ -208,7 +211,10 @@ int main(int argc, char **argv)
 	int disallow_selfsigned = 0;
 #endif
 
+#ifdef WIN32
```

Comment:
Fixed by removing #ifdef around definition and initialization of "debug_level" and moving the #endif that was behind the call to lws_set_log_level() in front of it (just like it already the case in test-server.c).

Also replaced all WIN32 occurrences by _WIN32 in test-echo.c, test-ping.c, and test-server.c as requested.